gradio_tts_app.py

```
@@ -272,103 +299,134 @@ def load_and_prepare_model():
+    return np.array([])
 
-    text_chunks = split_text_into_chunks(text)
-    logger.info(f"   Processing {len(text_chunks)} text chunk(s)")
 
-    generated_wavs = []
-    for i, chunk in enumerate(text_chunks):
-        logger.info(f"   Chunk {i+1}/{len(text_chunks)}: '{chunk[:50]}...'")
-        chunk_start = time.time()
+def generate(text: str, ref_wav: str, generation_config: GenerationConfig, chunking_config: ChunkingConfig, seed_num: int):
+    """Generate audio with improved device management and performance"""
+    global GLOBAL_MODEL
```

Flake8 (7.2.0)

[error] `global GLOBAL_MODEL` is unused: name is never assigned in scope

(F824)